pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2029,7 +2029,13 @@
       }
     },
     "System.IO.Ports": {
-      "InboxOn": {},
+      "StableVersions": [
```

Comment:
We shouldn't be adding stable versions unless we actually shipped this before.  It looks like we did ship this before but the UWP branch is missing the [update](https://github.com/dotnet/corefx/pull/22704) that @weshaggard made to master.

Given that we're missing the updates for all the other libraries I'd avoid adding these for ports.  If we do plan to ship any library packages out of this branch then we need to bring wes's change to this branch.